code/game/objects/items/eightball.dm

```diff
@@ -39,7 +39,7 @@
 /obj/item/toy/eightball/Initialize(mapload)
 	..()
```

Comment:
. = ..()

Reply:
#30831

Reply:
Cyberboss I don't understand your comment, could you be more descriptive? Just say why you wont' change it